group-coordinator/src/main/java/org/apache/kafka/coordinator/group/GroupConfig.java

```diff
@@ -147,4 +167,11 @@ public int sessionTimeoutMs() {
     public int heartbeatIntervalMs() {
         return consumerHeartbeatIntervalMs;
     }
+
+    /**
+     * The share group record lock duration in milliseconds
+     */
+    public int recordLockDurationMs() {
```

Comment:
Really, calling this `shareRecordLockDurationMs` would be preferable, and you should rename the consumer group ones to start with `consumer` such as `consumerHeartbeatIntervalMs`.